userguide/account.rst

```diff
@@ -168,46 +166,33 @@ operating system, as shown in
    Managing User Accounts
 
 
-Each account entry indicates the user ID, username, primary group ID,
-home directory, default shell, full name, whether it is a
-built-in user that came with the %brand% installation, the email
-address, whether logins are disabled, whether the user
-account is locked, whether the user is allowed to use
-:command:`sudo`, and if the user connects from a Windows
-8 or higher system. To reorder the list, click the desired column
-name. An arrow indicates which column controls the view sort order.
-Click the arrow to reverse the sort order.
+Each account entry indicates the username, user ID, primary group ID,
+home directory, default shell, and whether it is a built-in user that
+came with the %brand% installation. To reorder the list, click the
+desired column name. An arrow indicates which column controls the view
+sort order. Click the arrow to reverse the sort order.
 
-Click a user account to cause these buttons to appear:
+Highlight a user account and click :guilabel:`More Options` to cause
+the :guilabel:`Edit` and :guilabel:`Delete` buttons to appear.
 
-* **Modify User:** used to modify the account's settings, as listed
-  in :numref:`Table %s <user_account_conf_tab>`.
-
-* **Change E-mail:** used to change the email address associated with
-  the account.
-
-.. note:: It is important to set the email address for the built-in
+.. note:: It is recommended to set the email address for the built-in
    *root* user account as important system messages are sent to the
    *root* user. For security reasons, password logins are disabled for
    the *root* account and changing this setting is highly discouraged.
 
+
 Except for the *root* user, the accounts that come with %brand%
 are system accounts. Each system account is used by a service and
 should not be used as a login account. For this reason, the default
 shell on system accounts is
 `nologin(8) <http://www.freebsd.org/cgi/man.cgi?query=nologin>`__.
-For security reasons, and to prevent breakage of system services, do
+For both security reasons and to prevent breakage of system services, do
```

Comment:
This reads like there are two security reasons.  The original was clearer, although the comma can be removed.